src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -1384,6 +1384,53 @@ public void overdueFineShouldBeChargedWhenItemIsOverdue() throws InterruptedExce
     assertThat("item ID is included", account.getString("itemId"), UUIDMatcher.is(nod.getId()));
   }
 
+  @Test
+  public void overdueFineShouldNotBeChargedWhenShouldBeForgiven() throws InterruptedException {
+    IndividualResource loanPolicy = loanPoliciesFixture.create(
+      new LoanPolicyBuilder().rolling(Period.from(10, "Days")));
+
+    useFallbackPolicies(loanPolicy.getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId(), // forgiveOverdueFine = true
```

Comment:
Similar to the other comment. If `forgiveOverdueFine` being true is important to the test scenario, please define it explicitly in the test rather than using a comment.

Reply:
Done